src/component/canvas/ScatterRenderable.jsx

```diff
@@ -129,7 +127,9 @@ export class ScatterRenderable extends React.PureComponent {
 
     render(): React.Element<any> {
         return <g>
-            {this.props.scaledData.map(this.buildDot)}
+            {this.props.scaledData
+                .filter(data => isNumber(data.x) && isNumber(data.y))
```

Comment:
the `data` arg should probably be `row` for consistency with the code above, otherwise all good